docs/specification/well-known-schema-formats.md

```diff
@@ -0,0 +1,33 @@
+## Well-known schema formats
+
+### proto
+
+- schema: .proto
+- schema_name: unspecified
+
+### proto-set
+
+- schema: [FileDescriptorSet](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/descriptor.proto)
+- schema_name: Fully qualified name to the message within the descriptor set.
+
+A DescriptorSet for the following .proto file, the fully qualified name for `Baz` is `foo.bar.Baz`.
+
+```
+package foo.bar;
+message Baz {}
+```
+
+### ros1msg
+
+- schema: Concatenated ROS1 .msg files
```

Comment:
"Concatenated" is not very specific – this is the `### MSG:` format right? Do we need to mention the `gendeps` thing or wherever that format is documented (lol)?

Reply:
This is the ### Format yes. Tragically I've not found docs for it?